### PR TITLE
copy over cow function to ottrpal

### PR DESCRIPTION
**Pull request by kweav**
:warning: this is a stacked PR based on #153 :warning:

As mentioned in code review for PR #153 and various issues (#137 and https://github.com/jhudsl/cow/issues/27) -- Tried to copy over the `cow::get_pages_url()` function so that it's usable without needing to import `cow`. 

I used `wget` to get the raw file for the `get_pages_url()` function from the `cow` repo and used `devtools::document()` after that. Removed references to `cow` within the `make_screenshots()` function.

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -23,6 +23,7 @@ export(get_gs_pptx)
 export(get_image_from_slide)
 export(get_image_link_from_slide)
 export(get_object_id_notes)
+export(get_pages_url)
```
Comment (kweav) [inline NAMESPACE:26]: Do I need to also add an `@export` to the `make_screenshots()` function?

File: R/get_pages_url.R [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#' @importFrom magrittr %>%
+#' @import dplyr
+#'
+#' @export
+#'
+#' @examples \dontrun{
+#'
+#' usethis::create_github_token()
+#'
+#' get_chapters("jhudsl/Documentation_and_Usability")
```
Comment (kweav) [inline R/get_pages_url.R:26]: Does this example need updated?

Comment (cansavvy) [reply R/get_pages_url.R:26]: Yeah that'd be good

**Review by cansavvy — APPROVED** [review]
Looks good to me! Maybe a minor update in the docs but otherwise it can be merged!

**Pull request merged**